test/legacy/getRootStoreTests.ts

```diff
@@ -0,0 +1,9 @@
+import 'jasmine';
+import { isObservableMap } from 'mobx';
+import getRootStore from '../../src/getRootStore';
+
+describe('getRootStore', () => {
+    it('returns an ObservableMap', () => {
+        expect(isObservableMap(getRootStore())).toBeTruthy();
+    });
+});
```

Comment:
It's useless anyway since rootStore is *typed* as an `ObservableMap`...so we're just testing typescript at this point.

---
In reply to: [127325138](https://github.com/Microsoft/satcheljs/pull/63#discussion_r127325138) [](ancestors = 127325138)